drivers/interrupt_controller/intc_gic.c

```diff
@@ -93,6 +93,22 @@ void arm_gic_eoi(unsigned int irq)
 	sys_write32(irq, GICC_EOIR);
 }
 
+void gic_raise_sgi(unsigned int sgi_id, uint64_t target_aff,
+		uint16_t target_list)
+{
+	uint32_t sgi_val;
+
+	ARG_UNUSED(target_aff);
+
+	sgi_val = GICD_SGIR_TGTFILT_CPULIST |
+		GICD_SGIR_CPULIST(target_list & GICD_SGIR_CPULIST_MASK) |
+		sgi_id;
+
+	__DSB();
+	sys_write32(sgi_val, GICD_SGIR);
+	__ISB();
```

Comment:
`dsb()` and `isb()`

Reply:
will update, thanks.

Reply:
Hi, @carlocaione ,

I replace __DSB with dsb() and __ISB() with isb(), there is no issue for ARM64, but there will be build warning on ARM platform, for example for board: mercury_xu:

drivers/interrupt_controller/intc_gic.c:107:2: warning: implicit declaration of function 'dsb' [-Wimplicit-function-declaration]
  107 |  dsb();
      |  ^~~
drivers/interrupt_controller/intc_gic.c:109:2: warning: implicit declaration of function 'isb' [-Wimplicit-function-declaration]
  109 |  isb();
      |  ^~~

but there is no such issue when we use __DSB() and __ISB(), so I would like to rollback this modification, what's your comments? thanks.

Jiafei.